src/state/actions/users.js

```diff
@@ -73,6 +73,9 @@ export const fetchUsers = () => {
 };
 
 const deleteLogo = oldLogo => {
+  if (!oldLogo.includes('firebasestorage')) {
```

Comment:
What is this?

Reply:
In case you want to delete a profile picture from a facebook/google user, you can not delete it from the storage because that user has the file on the provider storage instead of our firebase storage. That condition is for returning null in case the file is not saved into our storage bucket.